src/components/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import { useRef } from "react";
+
+const Button = ({children}) => {
+    const buttonRef = useRef(null);
+
+  function handleClick() {
+    window.alert(`You clicked ${buttonRef.current.textContent}`);
+  }
```

Comment:
You should stick with one pattern, you don't need to mix the function syntaxes, it can be good if you change this handleClick function to also be an arrow function.